README.md

```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Microverse capstone
+## Awesome books
```

Comment:
- [ ] In your README, kindly add instructions on how to set up and run the app locally.